### PR TITLE
Python: Async Events need an ID to work with Perfetto

### LINKED ISSUE #20: `id` should be a required parameter for async_events
From Google's Trace Format: 

```
Each async event has an additional required parameter id.  We consider the events with the same category and id as events from the same event tree. An optional scope string can be specified to avoid id conflicts, in which case we consider events with the same category, scope, and id as events from the same event tree.
```

### PR DESCRIPTION
**Pull request by tpatki**
This PR fixes a bug and sets a default ID to 123 when it is not set.
Closes #20.

### CODE REVIEW

**Comment by slabasan** [comment]
@tpatki Do we need to set a default on the C side as well?

**Comment by tpatki** [comment]
Yes we do need it in C side too, let me check what's happening on the C side and if it is already addressed.

File: src/python/perfflowaspect/advice_chrome.py [hunk 1]
```diff
@@ -225,6 +225,8 @@ def create_async_event(cls, name, cat, ph, scope=None, id=None):
         event["ph"] = ph
         if id:
             event["id"] = id
+        else:
+            event["id"] = "123" #Async events need an ID, this sets the default.
```
Comment (slabasan) [inline src/python/perfflowaspect/advice_chrome.py:229]: Does this need to be a string or should it be an int (no quotes)?

Comment (tpatki) [reply src/python/perfflowaspect/advice_chrome.py:229]: Good question, I don't know if it matters, but looks like we can go with an `int`. I had picked that up from Dong's code somewhere I think. 

Looking at the format doc, Page 8, they seem to be using a hex ID: https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#

**Review by slabasan — APPROVED** [review]
LGTM

**Pull request merged**